src/config/dynamic-theme-fixes.config

```diff
@@ -3603,6 +3603,13 @@ CSS
 
 ================================
 
+dell.com
+
+INVERT
+svg
```

Comment:
Ehhh... All SVG's? That's pretty optimistic.

Reply:
See yourself. 😁

Reply:
It also inverts the icon on the top-right corner, which is are already correct... Ehm what about `.hpg-cat-item-wrap svg`